elements/rh-progress-stepper/rh-progress-step.css

```
@@ -0,0 +1,148 @@
+/* Main step container styling */
+*,
+*:after,
+*:before {
+  box-sizing: border-box;
+}
+
+:host {
+  --_step-line-style:
+    linear-gradient(
+        var(--_step-line-angle),
+        transparent 13px,
+        var(--_step-line-color) 13px,
+        var(--_step-line-color) 15px,
+        transparent 15px);
+
+  container-type: inline-size;
+
+  &:has(+ :not([state='inactive'], :not([state]))) {
+    --_step-line-color: var(--rh-color-border-strong) !important;
+  }
+}
+
+#container {
+  display: grid;
+  grid-template-columns: var(--_step-template-columns);
+  grid-template-rows: var(--_step-template-rows, 24px) auto;
+  grid-template-areas: var(--_step-template-areas);
+  gap: var(--_step-row-gap, 16px) var(--_step-column-gap, 16px);
+  justify-self: var(--_step-justify-self, start);
+  align-items: var(--_step-align-items);
+  padding-block-end: var(--_step-compact-vertical-padding);
+  padding-inline-end: var(--_step-compact-horizontal-padding);
+  width: 100%;
+  height: 100%;
```

```suggestion
  inline-size: 100%;
  block-size: 100%;
```

Logical properties FTW.